### PR TITLE
Product Subscriptions: Release i1 of feature and update icon

### LINKED ISSUE #11197: Update bottom sheet icons p1700210181490559/1699617986.069299-slack-C03L1NF1EA3

### PR DESCRIPTION
**Pull request by hafizrahman**
Part of https://github.com/woocommerce/woocommerce-ios/issues/11090
Closes https://github.com/woocommerce/woocommerce-ios/issues/11197

## Description

This PR:
- enables the subscription product feature flag so it's released to merchants
- updates the icon for "Simple subscription product" and "Variable subscription product" in the product creation bottom sheet, to match Android. It should now use this icon: https://fonts.google.com/icons?icon.query=event+repeat&icon.platform=ios

## Testing instructions

1. Check icon as above
2. Nothing else.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Resources/Images.xcassets/icon-subscription-product.symbolset/icon-subscription-product.svg
Comment (hafizrahman) [inline WooCommerce/Resources/Images.xcassets/icon-subscription-product.symbolset/icon-subscription-product.svg:1]: This is the file I got automatically from downloading on https://fonts.google.com/icons?icon.query=event+repeat&icon.platform=ios

Seems perhaps an overkill with different versions/thickness, but couldn't figure out how to just pick one.

Comment (hafizrahman) [reply WooCommerce/Resources/Images.xcassets/icon-subscription-product.symbolset/icon-subscription-product.svg:1]: Updated the icon(s) with different items from SF Symbols to match design from Joe.

I updated the screenshot in the first post above to reflect this.

File: Experiments/Experiments/DefaultFeatureFlagService.swift [hunk 1]
```diff
@@ -85,7 +85,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .customLoginUIForAccountCreation:
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .subscriptionProducts:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```
Comment (selanthiraiyan) [inline Experiments/Experiments/DefaultFeatureFlagService.swift:88]: Leaving a note for future reference. We should be removing the feature flag totally, which I have taken care of in this PR https://github.com/woocommerce/woocommerce-ios/pull/11248

**Review by selanthiraiyan — APPROVED** [review]
Thank you for taking care of the icons, Hafiz! This looks great. 🎉

**Pull request merged**